ddtrace/contrib/aiobotocore/patch.py

```diff
@@ -21,7 +22,7 @@ def patch():
     setattr(aiobotocore.client, '_datadog_patch', True)
 
     wrapt.wrap_function_wrapper('aiobotocore.client', 'AioBaseClient._make_api_call', _wrapped_api_call)
-    Pin(service='aws', app='aws', app_type='web').onto(aiobotocore.client.AioBaseClient)
+    Pin(service='aws', app='aws', tracer=tracer, app_type='web').onto(aiobotocore.client.AioBaseClient)
```

Comment:
Not sure it will work. When you import `tracer`, you have a local copy (the reference) in this module. It means if something changes `ddtrace.tracer` at runtime (i.e. you patch and initialize the OpenTracing Tracer), here you'll sending traces to the previous Tracer.

it must be `ddtrace.tracer` all the time, but it must be evaluated at runtime, and not set at the beginning when patching, otherwise you'll have the same problem.

Reply:
Oh I see. Of course we’ll be setting `ddtrace.tracer` at runtime. I guess we could instead do an in-line:

```
import ddtrace
Pin(...,tracer=ddtrace.tracer,...)
```

instead? Then we’ll get the most recent reference.